examples/non-fungible-token/tests/workspaces/test_approval.rs

```diff
@@ -11,10 +13,12 @@ pub const TOKEN_ID: &str = "0";
 const ONE_NEAR: NearToken = NearToken::from_near(1);
 const ONE_YOCTO: NearToken = NearToken::from_yoctonear(1);
 
+#[rstest]
 #[tokio::test]
-async fn simulate_simple_approve() -> anyhow::Result<()> {
-    let worker = near_workspaces::sandbox().await?;
-    let (nft_contract, alice, token_receiver_contract, _) = init(&worker).await?;
+async fn simulate_simple_approve(
+    #[future] initialized_contracts: anyhow::Result<(Contract, Account, Contract, Contract)>,
+) -> anyhow::Result<()> {
+    let (nft_contract, alice, token_receiver_contract, _) = initialized_contracts.await?;
```

Comment:
Do we win much from using fixtures here? It took me several minutes to figure out and I think it is unnecessary magical, and makes the code less readable. Also, rstest dependency might add extra compilation time.

Reply:
such a minimal manifest with `rstest` with a corresponding test
```toml
[package]
name = "rstest_play"
version = "0.1.0"
edition = "2021"

[dependencies]

[dev-dependencies]
rstest = "0.23.0"
```
compiles for 
```bash
   Compiling rstest v0.23.0
    Finished `test` profile [unoptimized + debuginfo] target(s) in 4.86s
```
on my machine, which would probably mean 10-15 secs on gh executor